crates/bevy_ecs/src/query/iter.rs

```diff
@@ -1076,11 +1077,17 @@ where
                 continue;
             }
 
-            let archetype = self
-                .archetypes
-                .get(location.archetype_id)
-                .debug_checked_unwrap();
-            let table = self.tables.get(location.table_id).debug_checked_unwrap();
+            let (archetype, table);
```

Comment:
This is good cleanup, but could probably be a separate PR.

Reply:
Fair, I'll put it in a different PR.